artemis-junit/artemis-junit-5/pom.xml

```diff
@@ -57,12 +57,7 @@
       </dependency>
       <dependency>
          <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-jms-server</artifactId>
-         <version>${project.version}</version>
-      </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-jms-client</artifactId>
```

Comment:
The JUnit infrastructure should be using the core API exclusively so I don't think there's any reason to depend on the core JMS client.

Reply:
I was 'ok' with this removal since the junit 5 stuff is actually 'core centric' (which again, I dont actually think makes any sense, since we would recommend the JMS client to most, and most I've seen using this want it to test with the JMS client, so they typically shouldn't be using the core API directly for anything).

Thats unlike the original junit 4 aimed stuff though, which has some very JMS-specific bits as well, that do use the JMS client, so I dont think it should be removed from the junit 4 stuff at this point since that has always supplied the JMS client, and some will expect it to.

I have however been forgetting about junit-commons module also existing, which is perhaps still supplying the jms client to _both_ the junit 4 and 5 modules (the latter should arguably be excluding it if so) ?

Reply:
I see your point about the module providing the client to the depender to make testing easier. I think that's fair.